app/javascript/app/pages/custom-compare/custom-compare.js

```diff
@@ -17,19 +22,52 @@ const mapStateToProps = (state, { location, route }) => {
 
   return {
     anchorLinks: getAnchorLinks(routeData),
-    filtersData: getFiltersData(state, { search })
+    filtersData: getFiltersData(state, { search }),
+    selectedTargets: getSelectedTargets(state, { search }),
+    backButtonLink: getBackButtonLink(null, { search })
   };
 };
 
 const CustomCompare = props => {
   const { history } = props;
 
-  const handleFilterChange = (name, value) => {
-    const params = { name, value };
+  const updateTargetParams = newTargets => {
+    const params = {
+      name: 'targets',
+      value: newTargets.toString()
+    };
     history.replace(getLocationParamUpdated(location, params));
   };
 
-  return <Component {...props} handleFilterChange={handleFilterChange} />;
+  const handleCountryFilterChange = (targetKey, newCountry) => {
+    const { selectedTargets } = props;
+    const newTargetParams = [];
+    selectedTargets.forEach(({ key, country, document = '' }) => {
+      if (key === targetKey) newTargetParams.push(`${newCountry}-`);
```

Comment:
Shouldn't we always have a country and a document? Why add a country without a document?

Reply:
After changing the country we're not selecting automatically the document, we leave the dropdown empty. The question is what do we want to do with that @fannycc @Bluesmile82?
![vaq2b-z5etk](https://user-images.githubusercontent.com/15097138/75331140-1fbdac80-587a-11ea-8503-07c0d49a1198.gif)